### DISCUSSION TITLE
Welcome to wikmd Discussions!

### DISCUSSION THREAD

**Discussion by Linbreux** [Linbreux/wikmd#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mephue** [comment]
Hello, I have found a few vulnerabilities in your application which I would like to share with you so that they can be fixed. Can you provide a mail address or create a security policy?
